scripts/install-jstz-cli.sh

```diff
@@ -38,7 +39,13 @@ jstz_configure() {
         mkdir -p "$jstz_home"
         cat >"$jstz_home/config.json" << EOF
 {
-  "default_network": "dev"
+  "default_network": "dev",
+  "networks": {
+    "weeklynet": {
+      "octez_node_rpc_endpoint": "http://rpc.$network.teztnets.com",
+      "jstz_node_endpoint": "http://34.39.12.211:8933"
+    }
+  }
 }
 EOF
 
```

Comment:
The manual alias and shell alias differ

Reply:
Will fix now

Reply:
Fixed 9169132759c687f3ca26de7139ffdc820c3ea119